packages/api/src/typeDefs/index.ts

```diff
@@ -21,29 +21,33 @@ import Seo from './seo.graphql'
 import Cart from './cart.graphql'
 import Status from './status.graphql'
 import PropertyValue from './propertyValue.graphql'
+import VTEXQuery from '../platforms/vtex/customTypeDefs/query.graphql'
```

Comment:
How about
```suggestion
import VTEXQueries from '../platforms/vtex/customTypeDefs/query.graphql'
```
?
Since we might have more in the future.

Reply:
Fixed